config/config.go

```diff
@@ -28,6 +28,9 @@ type MongoConfig struct {
 	BindAddr   string `envconfig:"MONGODB_BIND_ADDR"   json:"-"`
 	Collection string `envconfig:"MONGODB_COLLECTION"`
 	Database   string `envconfig:"MONGODB_DATABASE"`
+	Username   string `envconfig:"USERNAME"`
+	Password   string `envconfig:"PASSWORD"`
```

Comment:
These environment variables are ambiguous, I'd suggest prefixing them with `MONGODB_` like the previous ones.
```suggestion
	Username   string `envconfig:"MONGODB_USERNAME"`
	Password   string `envconfig:"MONGODB_PASSWORD"`
```

Reply:
made the changes. thanks for the review.